fixtures/bench_accumulator_factory.bash

```diff
@@ -1,18 +0,0 @@
-#!/bin/sh
```

Comment:
Any reason to remove this file? Is it the one that triggers a bug? If so, I propose renaming it to `.bash.bug` or something like this. Tests will skip it.

Reply:
Yes, this is the one triggering the bug. I've just readded it, with a comment indicating the problem and a link to the upstream issue.